omod/src/main/web/dashboardwidgets/latestobsforconceptlist/latestobsforconceptlist.spec.js

```diff
@@ -30,9 +30,8 @@ describe('LatestObsForConceptList', () => {
 		let bindings = {config: { maxAge: '2w', concepts: 'uuid-1, uuid-2', patientUuid: 'patientUuid' }};
 		let ctrl = $componentController('latestobsforconceptlist', {$scope}, bindings);
 		
-		$httpBackend.expectGET('/ws/rest/v1/obs?concept=uuid-1&patient=patientUuid&v=full').respond({results: []});
```

Comment:
Did you intentionally remove these two lines?

Reply:
@dkayiwa Yes I did since the message signature has changed to support two concepts, but now that you mention it, I should have just added the third test since the other two should still work.

Reply:
I have added a test case for a single concept instead